### DISCUSSION TITLE
Welcome to Cleftly Discussions!

### DISCUSSION THREAD

**Discussion by wxllow** [cleftly/cleftly#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Please feel free to introduce yourself below :\)

**Comment by skym-t** [comment]
Nice

**Comment by wxllow** [comment]
> Nice

damn i didnt realize u were here hi
